docs/Tutorials/apiTutorial.md

```diff
@@ -21,158 +21,134 @@ A minimum network configuration must include:
-        })
-        
-        req.on('error', (e) => {
-            console.error(e)
-        })
-        
-        req.end()
+1. Open a Terminal or Command Prompt.
+2. Copy and paste the example code below into a text editor.
+3. Update the values of the `[IP-Address or FQDN]` and `[JWT-Token]` fields.
 
+    === "Linux"
+        ```bash
+        curl --insecure https://[IP-Address or FQDN]/mps/api/v1/devices \
+            -H "Authorization: Bearer [JWT-Token]"
+        ```
+    === "Windows"
+        ```bash
+        curl --insecure https://[IP-Address or FQDN]/mps/api/v1/devices ^
+            -H "Authorization: Bearer [JWT-Token]"
         ```
 
-4. Run the code snippet using node.
-
-    ```
-    node myDevices.js
-    ```
-
-    !!! important
-        This is one way to retrieve a device's GUID in the *host* field.  **For *amt* path methods (i.e., [Power Actions](../Topics/powerstates.md), Audit Logs, etc), the device GUID is *required* as part of the GET path.** Save this value if you want to try other MPS methods. Other ways to retrieve a GUID can be found [here](../Topics/guids.md).
-
+4. Run the command.
 
-    !!! example
+    !!! example "Example - Response of Devices Method"
         Example Terminal Output:
 
         ```json
-        [{"connectionStatus":1,"hostname":"DESKTOP-R2225SQ",    "guid":"d92b3be1-b04f-49de-b806-54b203054e9d","metadata":   {"guid":"d92b3be1-b04f-49de-b806-54b203054e9d","hostname":"DESKTOP-R2225SQ","tags":    []}}]
+        [{"guid":"3beae094-34f8-11ea-b6f5-ffed08129200","hostname":"vpro3-NUC8v5PNK","tags":[],"mpsInstance":"mps","connectionStatus":true,"mpsusername":"admin"}]
         ```
         Example JSON Pretty Print:
 
         ```json
         [
             {
-                "connectionStatus": 1,
-                "hostname": "DESKTOP-R2225SQ",
-                "guid": "d92b3be1-b04f-49de-b806-54b203054e9d",
-                "metadata": {
-                    "guid": "d92b3be1-b04f-49de-b806-54b203054e9d",
-                    "hostname": "DESKTOP-R2225SQ",
-                    "tags": []
-                }
+                "guid": "3beae094-34f8-11ea-b6f5-ffed08129200",
+                "hostname": "vpro3-NUC8v5PNK",
+                "tags": [],
+                "mpsInstance": "mps",
+                "connectionStatus": true,
+                "mpsusername": "admin"
             }
         ]
         ```
+    
+        !!! important
+            This is one way to retrieve a device's GUID in the *host* field.  **For *amt* path methods (i.e., [Power Actions](../Topics/powerstates.md), Audit Logs, etc), the device GUID is *required* as part of the GET path.** Save this value if you want to try other MPS methods. Other ways to retrieve a GUID can be found [here](../Topics/guids.md).
+
+
+
+## Example GET/POST Templates
+
+The sample GET and POST curl commands below can be adapted for other MPS and RPS methods **by changing the URI path and modifying the request body data, if applicable**.
```

Comment:
Do you mean URL here, or is it a considered a URI in this context? Typo or intentional?